airflow/models/baseoperator.py

```diff
@@ -1121,6 +1137,12 @@ def set_upstream(self, task_or_task_list: Union['BaseOperator', List['BaseOperat
         """
         self._set_relatives(task_or_task_list, upstream=True)
 
+    @property
+    def output(self):
```

Comment:
are we all agree on the name? Just raising it for a discussion - xcom_output?

Reply:
Rather avoid making xcom explicit here. Ideally XCom can change underneath to some other class for communication betwen operations (changing the xcom_class in airflow.cfg?). Also for simplicity `output` is shorter to write in a DAG 😄

Reply:
I think it's more in line with already existing `outlet` and `inlet`